README.md

```diff
@@ -1,85 +1,67 @@
-# Membership
-
-The Node.js Foundation has an Individual Membership Program.
-
-The Foundation is a member based organization. Previously the
-foundation only had the ability for companies and non-profits
-to be members, but as of November 2015, the Foundation also
-accepts [individual humans as members][4].
-
-Individual members of the Node Foundation have the ability to
-both run and vote for representation on the [Board of Directors][5].
-
-## 2017 Election
-
-Nominations for the 2017 Election were posted [here][11].
-
-The winner of the 2017 election will serve a two year term. They will join 
-Ashley Williams [@ashleygwilliams](https://github.com/ashleygwilliams) who will
-serve with the elected director in 2017 for the balance of the 2 year term.
-
-For more information about Individual Membership on the Board,
-see below.
-
-## 2016 Election
-
-Nominations for the 2016 Election were posted [here][1].
-
-The 2016 election was the first Individual Membership Election. The winner of the 
-election is serving a two year term and the runner-up is serving a one year term.
-Future elections will happen each year for a new two year term.
-
-- The board members elected in the 2016 election are:
-  - Ashley Williams [@ashleygwilliams](https://github.com/ashleygwilliams) for a 2 year term.
-  - Feross Aboukhadijeh [@feross](https://github.com/feross) for a 1 year term.
-
-## Cost
-
-- Node.JS Project Contributors (any member of the GitHub Org): [__free__][2]*
+# Board of Directors
+
+The Node.js Foundation is a member based organization. There are 5 classes of Membership: 
+Platinum, Gold, and Silver Members, Associated Entity Members and Individual Members.
+
+A list of non-Individual members and additional information is available on the website:
```

Comment:
@jasnell that's not what they area. That's a colloquialism. The page lists Platinum Members, Gold Members, Silver Members, and Associated Entity Members. Would you rather I write just list all of those?